### ISSUE TITLE
Bug detected in `getSubsetEcModel`

### ISSUE THREAD

**Issue by mihai-sysbio** [SysBioChalmers/GECKO#327]
### Description of the bug:

Initially spotted [here](https://github.com/SysBioChalmers/GECKO/pull/322), running the last step of the light tutorial triggers an error.

#### Reproducing these results:

```matlab
>> ecHT29 = getSubsetEcModel(ecModel,HT29);
Matrix index is out of range for deletion.

Error in removeReactions (line 41)
        reducedModel.rxns(indexesToDelete)=[];

Error in getSubsetEcModel (line 53)
smallEcModel = removeReactions(smallEcModel,~keepRxns, true, true, true);
```

#### System information

* Operating system: macOS 13 
* MATLAB version: 2021b 
* GECKO version: develop 

**I hereby confirm that I have:**
- [x] Tested my code with [all requirements](https://github.com/SysBioChalmers/GECKO) for running GECKO
- [x] Done this analysis in the `develop` branch of the repository
- [x] Checked that a similar issue does not exist already

**Comment by mihai-sysbio** [comment]
These two lines are identical 
https://github.com/SysBioChalmers/GECKO/blob/38b6768b2ef9fb801a6739ef653372c3766c9f5a/src/geckomat/utilities/getSubsetEcModel.m#L41
https://github.com/SysBioChalmers/GECKO/blob/38b6768b2ef9fb801a6739ef653372c3766c9f5a/src/geckomat/utilities/getSubsetEcModel.m#L53

but I don't understand how come there is no 

```matlab
smallEcModel = removeReactions(bigEcModel,~keepRxns, true, true, true);
```

**Comment by edkerk** [comment]
Line 53 should be deleted.



> These two lines are identical
> 
> https://github.com/SysBioChalmers/GECKO/blob/38b6768b2ef9fb801a6739ef653372c3766c9f5a/src/geckomat/utilities/getSubsetEcModel.m#L41
> 
> 
> https://github.com/SysBioChalmers/GECKO/blob/38b6768b2ef9fb801a6739ef653372c3766c9f5a/src/geckomat/utilities/getSubsetEcModel.m#L53

Line 53 should be deleted.

> 
> but I don't understand how come there is no
> 
> ```matlab
> smallEcModel = removeReactions(bigEcModel,~keepRxns, true, true, true);
> ```

That would not make sense. The smallEcModel is starting to get generated in line 26, and then modified in the rest of the function.

https://github.com/SysBioChalmers/GECKO/blob/38b6768b2ef9fb801a6739ef653372c3766c9f5a/src/geckomat/utilities/getSubsetEcModel.m#L26

**Comment by edkerk** [comment]
Resolved in #331
